web/client/utils/EncodeUtils.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Convert an UTF-8 into a byte representation. It usefult to avoid the `btoa` Unicode problem
+ * A more detailed explaination about this can be found here:
```

Comment:
here where?

Reply:
this one 
https://developer.mozilla.org/en-US/docs/Glossary/Base64#the_unicode_problem 
?

Reply:
yes I missed the link 
```suggestion
 * A more detailed explaination about this can be found here: https://developer.mozilla.org/en-US/docs/Glossary/Base64#the_unicode_problem
```